include/deal.II/base/parameter_handler.h

```diff
@@ -1558,7 +1558,35 @@ class ParameterHandler : public Subscriptor
     ShortText = 193
   };
 
+  /**
+   * Flags to control input file scanning behavior.
+   */
+  enum ReadInputFlags
+  {
+    /**
+     * A flag for no flag.
+     */
+    ReadInputFlag_Null = 0,
 
+    /**
+     * If this flag is set, the read_input function will automatically generate
+     * the requested file with default values if the provided file does not exist.
+     */
+    ReadInputFlag_InputFileHelp = 0x0001,
+
+    /**
+     * If this flag is set and the "Optinal" option is actived, the automatically
+     * generated file will not contain additional comments.
+     */
+    ReadInputFlag_WriteCompact = 0x0002,
```

Comment:
Referring to the optional flag doesn't make sense in the documentation here, instead you should mention the flag above.